internal/commands/servergroup/list.go

```diff
@@ -0,0 +1,47 @@
+package servergroup
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud/request"
+)
+
+// ListCommand creates the "servergroup list" command
+func ListCommand() commands.Command {
+	return &listCommand{
+		BaseCommand: commands.New("list", "List current server groups", "upctl servergroup list"),
+	}
+}
+
+type listCommand struct {
+	*commands.BaseCommand
+}
+
+// ExecuteWithoutArguments implements commands.NoArgumentCommand
+func (s *listCommand) ExecuteWithoutArguments(exec commands.Executor) (output.Output, error) {
+	svc := exec.All()
+	clusters, err := svc.GetServerGroups(exec.Context(), &request.GetServerGroupsRequest{})
+	if err != nil {
+		return nil, err
+	}
+
+	rows := []output.TableRow{}
+	for _, serverGroup := range clusters {
+		rows = append(rows, output.TableRow{
+			serverGroup.UUID,
+			serverGroup.Title,
+			serverGroup.AntiAffinityPolicy,
+		})
+	}
+
+	return output.Table{
```

Comment:
Hmm, would the JSON and YAML pass-through work here as well (similarly than with `show` command)? 🤔 We don't use that in other lists either, but script usage might benefit from those omitted fields.

Reply:
Thanks for the suggestion, now implemented! I'll make a note to add the pass-through to other resource lists as well (where applicable).